client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -471,6 +473,7 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
 				</div>
 			}
 			recordTracksEvent={ recordTracksEvent }
+			goBack={ () => submit?.( { previousStep: true } ) }
```

Comment:
This seems a little hacky.

Given we have access to `isCopySiteFlow( flow )` in this context, can we call `window.location.assign( '/sites' )` directly here?

Reply:
It's a bit hacky, but I don't think that adding `window.location.assign` here is the best way to go. All the navigation decisions are happening inside `useStepNavigation` which is why the function exists. Adding it here, we scatter the navigation decision. There is a `goBack` function we can use, but the linter marks it as it's deprecated.

Reply:
- This is done, after merging https://github.com/Automattic/wp-calypso/pull/72450